dynawo/sources/Solvers/AlgebraicSolvers/DYNSolverKINAlgRestoration.h

```diff
@@ -43,7 +43,7 @@ class SolverKINAlgRestoration : public SolverKINCommon, private boost::noncopyab
   /**
    * @brief default constructor
    */
-  SolverKINAlgRestoration();
+  explicit SolverKINAlgRestoration(bool printReinitResiduals);
```

Comment:
const bool to be coherent with the implementation file

Reply:
yes it does compile as shown by the CI but it is not good for code readability. Anyway, not a big deal.